pkg/grpc/clientGRPC.go

```diff
@@ -0,0 +1,47 @@
+package grpc
+
+import (
+	"context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	pb "monorepa/pkg/grpc/proto"
+	"monorepa/pkg/storage"
+)
+
+// Client GRPC
+
+type gRPCClient struct {
+	client pb.GrpcServiceClient
+}
+
+func new(addr string) *gRPCClient {
```

Comment:
Where you use it???

Reply:
It was used only for testing

Reply:
Don`t see any test with it. Look for unused code.

Reply:
Without this file my tests falls, because test file used func getItems from client side, but i can delete this file if it is necessary